web/packages/teleterm/src/ui/Search/pickers/ResultList.tsx

```diff
@@ -145,18 +145,24 @@ export const NonInteractiveItem = styled.div`
 
   padding: ${props => props.theme.space[2]}px;
   color: ${props => props.theme.colors.text.contrast};
-  background: ${props =>
-    props.$active
-      ? props.theme.colors.levels.elevated
-      : props.theme.colors.levels.surface};
+  background: ${props => props.theme.colors.levels.surface};
 `;
 
 const InteractiveItem = styled(NonInteractiveItem)`
+  cursor: pointer;
+
   &:hover,
   &:focus {
-    cursor: pointer;
     background: ${props => props.theme.colors.levels.elevated};
   }
+
+  ${props => {
+    if (props.active) {
+      return css`
+        background: ${props => props.theme.colors.levels.elevated};
```

Comment:
This is the CSS that ends up being generated. It shouldn't cause problems here but I'd be wary of using this approach in general.

![generated CSS](https://user-images.githubusercontent.com/27113/232014977-ee97972b-0d64-4b86-a9ef-2bd405998f11.png)